docs/content/src/develop/managing-ballerina-code.md

```diff
@@ -0,0 +1,352 @@
+# Managing Ballerina Code
+
+When you write large programs often you would want to organize your code into shareable units. 
+In Ballerina, you can divide your program into such units which are called Ballerina modules. 
+Ballerina modules reside within a Ballerina project. You can have multiple modules within a 
+Ballerina project. 
+
+To create a project you can use the WSO2 Ballerina VS Code plugin or the Ballerina CLI tool.
+
+## Creating a project
+
+To create a new Ballerina project with Ballerina CLI tool use `ballerina new`:
+
+```bash
+$ ballerina new hello-world
+```
+
+Let's see what Ballerina CLI tool has generated for us.
+
+```bash
+$ cd hello-world
+$ tree .
+.
+├── Ballerina.toml
+└── src
+
+1 directory, 1 file
+
+```
+
+If we take a look at the `Ballerina.toml` file
+
+```toml
+[project]
+org-name= "wso2"
+version= "0.1.0"
+
+[dependencies]
+
+```
+
+This is a manifest file. This file is used by the compiler to identify a ballerina project directory. 
+Please note that the `B` is capital in Ballerina.toml file.
+
+We will get into more details about `Ballerina.toml` file as we progress through this guide.
+
+## Ballerina Modules
+
+Within the `src` directory of a Ballerina project, you can have multiple directories. Each directory 
+will be treated as a Ballerina module.
+
+A module is a directory that contains Ballerina source code files and is part of a namespace. Modules 
+facilitate collaboration, sharing, and reuse. Modules can include functions, connectors, constants, 
+annotations, services, and objects. To share a module among programs, projects, and users you need to 
+push the module into a repository like Ballerina Central.
+
+> **Tip**: Module names can contain alphanumeric characters including dots `.`. Dots in a module name 
+> have no meaning other than the last segment after the final dot being used as a default alias within 
+> your source code.
+
+To create a Ballerina Module you can simply,
+
+- create a directory within the src directory 
```

Comment:
```suggestion
To create a Ballerina module, create a directory within the `src` directory and do one of the following: 
```

Reply:
`- create a directory within the src directory ` is an option too. So we can't merge the lines, Shall I put an or after that line?

Reply:
Maybe better to say something like "To create a Ballerina module, do one of the following:". After that we can introduce the list without using "or" at all.